op-service/sources/types.go

```diff
@@ -61,6 +61,9 @@ type RPCHeader struct {
 	// ParentBeaconRoot was added by EIP-4788 and is ignored in legacy headers.
 	ParentBeaconRoot *common.Hash `json:"parentBeaconBlockRoot,omitempty"`
 
+	// RequestsHash was added by EIP-7685 and is ignored in legacy headers.
+	RequestsHash *common.Hash `json:"requestsHash" rlp:"optional"`
```

Comment:
```suggestion
	RequestsHash *common.Hash `json:"requestsHash,omitempty" rlp:"optional"`
```

Reply:
Good catch

Reply:
Good spot -- I see we are doing that on a bunch of other fields on this struct. Is there a reason why we are doing this when geth doesn't? https://github.com/ethereum-optimism/op-geth/blob/c058cf26262bf35657a2990689a85af1a3bfe4c4/core/types/block.go#L108

Reply:
That's strange. I wouldn't expect them to serve older blocks with `null` attributes. Or `null` attributes for non-default features.

In the execution API spec it's not listed as required attribute: https://github.com/ethereum/execution-apis/blob/869b7f062830ba51a7fd8a51dfa4678c6d36b6ec/src/schemas/block.yaml#L4